scriptler/showJavaVersionUseOnJob.groovy

```diff
@@ -0,0 +1,32 @@
+/*** BEGIN META {
+  "name" : "Show Java Version of job",
+  "comment" : "Show Java Version used on job",
+  "parameters" : [],
+  "core": "1.609",
+  "authors" : [
+    { name : "kuisathaverat" }
+  ]
+} END META**/
+
+import jenkins.model.Jenkins
+import hudson.model.AbstractProject
+import com.cloudbees.hudson.plugins.folder.Folder
+import hudson.triggers.TimerTrigger
+
+Jenkins jenkins = Jenkins.instance
+
+findAllItems(jenkins.items)
+
+def findAllItems(items){
+  for(item in items)
```

Comment:
I've already mentioned the functional alternatives enough :) You get the idea. Again, just a comment.